miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplay.kt

```diff
@@ -80,10 +79,16 @@ internal class RealMiniAppDisplay(
                 File(basePath)
             )
         )
+        .addPathHandler(
+            "/", WebViewAssetLoader.InternalStoragePathHandler(
+                context,
+                File(basePath)
+            )
+        )
         .build()
 
     @VisibleForTesting
-    internal fun getLoadUrl() = "$customScheme$SUB_DOMAIN_PATH/index.html"
+    internal fun getLoadUrl() = "$customDomain$SUB_DOMAIN_PATH/index.html"
```

Comment:
Now we again have this question of what should the HTTPS url be? Right now it would be something like `https://mscheme.MINI_APP_ID/miniapp/index.html`. Probably that's okay for now since we're not intending for users to use the HTTPS URL (they should use the custom scheme for redirecting to the mini app). But we'll need to change this URL in the future when we possibly replace the custom scheme on iOS with HTTPS.

Reply:
Yeah. In the past, we had `https://MINI_APP_ID.miniapps.androidplatform.net/miniapp/index.html` but this `androidplatform.net` is not required so it could be `https://miniapps.MINI_APP_ID/miniapp/index.html` 🤷‍♂️.
How do you think @ts-nitish-a-jain @corycaywood?

Reply:
Probably it's okay to keep as it is, and then we can update it in a later release (since we haven't yet decided what we're doing on iOS).

Reply:
Yeah, I think let's keep it `https://mscheme.MINI_APP_ID/miniapp/index.html` for now. Thanks @minh-rakuten @corycaywood